src/opnsense/mvc/app/models/OPNsense/Firewall/FieldTypes/AliasNameField.php

```diff
@@ -63,7 +63,7 @@ public function getValidators()
         $reservedwords = array(
             'all', 'allow-opts', 'altq', 'anchor', 'antispoof', 'any', 'bandwidth', 'binat', 'binat-anchor', 'bitmask',
             'block', 'block-policy', 'buckets', 'cbq', 'code', 'codelq', 'crop', 'debug', 'divert-reply', 'divert-to',
-            'drop', 'drop-ovl', 'dup-to', 'fail-policy', 'fairq', 'fastroute', 'file', 'fingerprints', 'flags',
+            'drop', 'drop-ovl', 'dup-to','dyninterface', 'fail-policy', 'fairq', 'fastroute', 'file', 'fingerprints', 'flags',
```

Comment:
@marjohn56 these are reserved pf words, you can remove your type from here.

Reply:
wasn't sure.... covering all bases. Will do and update.